File: examples/bench/int_loop.js
'use strict';
const { performance } = require('perf_hooks');

const N = 5000000;
const t0 = performance.now();
let sum = 0;
for (let i = 0; i < N; i++) {
  sum = sum + i;
}
const t1 = performance.now();
console.log('INNER_MS=' + Math.round(t1 - t0));
console.log('SUM=' + sum);

File: examples/bench/int_loop.py
import time

N = 5_000_000
t0 = time.perf_counter()
s = 0
for i in range(N):
    s = s + i
t1 = time.perf_counter()
print("INNER_MS=" + str(int(round((t1 - t0) * 1000))))
print("SUM=" + str(s))
